src/app/shared/issue-pr-card/issue-pr-card-labels/issue-pr-card-labels.component.ts

```diff
@@ -8,7 +8,7 @@ import { LabelService } from '../../../core/services/label.service';
   styleUrls: ['./issue-pr-card-labels.component.css']
 })
 export class IssuePrCardLabelsComponent {
-  @Input() labels: Label[];
-  @Input() labelSet: Set<Label>;
+  @Input() labels: GithubLabel[];
```

Comment:
Don't think this is needed now that the `name` property of a `Label` is present.

Reply:
Can I clarify, do you mean that the type annotation for the field `labels` can be left as `Label[]`? The reason I changed the type annotation is because in the only place that this component is used, in `issues-pr-card.component.html`:

```html
<mat-card-content>
  <app-issue-pr-card-milestone [milestone]="issue.milestone"></app-issue-pr-card-milestone>
  <app-issue-pr-card-labels [labels]="issue.githubLabels" [labelSet]="dropdownFilter?.hiddenLabels"></app-issue-pr-card-labels>
</mat-card-content>
```

The `labels` field is given `issue.githubLabels`, which is of type `GithubLabel[]`.

Reply:
Ok cool 🎉